SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -48,6 +48,9 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 	*/
 	virtual SpatialGDK::QueryConstraint GetWorkerInterestQueryConstraint() const PURE_VIRTUAL(UAbstractLBStrategy::GetWorkerInterestQueryConstraint, return {};)
 
+	/** True if this load balancing strategy requires handover data to be transmitted. */
+	virtual bool RequiresHandoverData() const { return false; }
```

Comment:
default on?

Reply:
I'm on the fence, leaning towards opt-in because of the performance implications. Not a strong opinion though.

Reply:
Perhaps this should be pure virtual, to force the user to decide on a case-by-case basis. I'm more concerned if they make their own LBS that uses zoning and forget to enable it.

Reply:
Good call.